src/angular/src/app/spark-core-angular/components/sprk-accordion/sprk-accordion.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'sprk-accordion',
+  template: `
+    <ul [ngClass]="getClasses()">
```

Comment:
i doubt it but wonder if they'd ever want to add analytics to the outer accordion here?